### DISCUSSION TITLE
✅ macOS Version

### DISCUSSION THREAD

**Discussion by brunoguirado** [isontheline/pro.webssh.net#149] [Ideas]
very usefull have a macos version. syncing hosts between ipad, iphone and mac 😍
 
today only termius has this, but not use icloud to sync, They  have a server and a terrible signature. 

will be a expressive diferencial

**Comment by isontheline** [comment]
Hello @brunoguirado 👋

macOS version planned 14 days ago : https://github.com/isontheline/pro.webssh.net/issues/116

Yes Termius uses it's own servers and it won't be WebSSH case.
iCloud is also planned here : https://github.com/isontheline/pro.webssh.net/issues/43

Have a nice day ☀️

**Comment by isontheline** [comment]
Hello @brunoguirado 👋

M1 support is almost done.
x86_64 will be the next step.

**Reply by NosTech** [reply]
Hi, I was wondering if there is some kind of target date for x86_64 support?
I really like the iOS app and was hoping to be able to use (and purchase) the app on macOS as well.

**Reply by isontheline** [reply]
Hello @NosTech 👋

I will publish an ETA as soon as possible.
The tracking issue is : https://github.com/isontheline/pro.webssh.net/issues/322#issuecomment-903169478
Please subscribe to this issue in order to stay tuned

**Reply by NosTech** [reply]
Thank you very much for your response. I have subscribed to the issue. Nice to see that you are still active working on the issue. Keep up the good work 👍🏻

**Comment by isontheline** [comment]
@brunoguirado are you using ARM ou X86_64 processor?

**Comment by isontheline** [comment]
WebSSH 14.16 is now available on the macOS : https://apps.apple.com/us/app/webssh-pro/id497714887

**Reply by cotchon** [reply]
Bonjour Arnaud,
I am testing the iCloud sync between devices, so I first tried to install the macos app.
Unfortunately I get this message : "L'article que vous cherchez n'est actuellement pas disponible dans le Store français"
Any clue ??
Merci :-)

**Reply by isontheline** [reply]
Bonjour à vous @cotchon 👋

Pour le moment il faut posséder un mac M1. Est-ce votre cas ?

L'application est disponible dans l'onglet "Apps pour iPad"

**Reply by cotchon** [reply]
Re,
Non j'ai un mac Intel sous high sierra
Cela doit être pour ça ;-) (j'ai mal lu juste au dessus)
J'attendrai les prochaines versions.
Merci !

**Reply by vamfoom** [reply]
Please give it the ability to import from ~/.ssh/config file also.

**Reply by isontheline** [reply]
@vamfoom You can track the progress here : https://github.com/isontheline/pro.webssh.net/issues/327

**Comment by isontheline** [comment]
WebSSH is now available on macOS since 16.0
macOS 11 is required.
Please open an issue for any glitches.

Have a nice day ☀️
